### ISSUE TITLE
Exploring Razorfen Kraul

### ISSUE THREAD

**Issue by Kehrer-Hellscream** [HellscreamWoW/Tracker#2995]
Razorfen Kraul in Southern barrens is not explorable, so the exploring-achievements southern barrens kalimdor -->world, are not available.

**Comment by Guybrush-Hellscream** [comment]
Yeah that one is really annoying, I couldn't manage to get this as well. Apparently, they have this problem on retail too. We might need to look on that.

**Comment by Collector-Hellscream** [comment]
#233 Already reported and extremely annoying ;d Also I don't think you can explore World currently. Vash'jir have some fatigue issues and to deepholm you can get only via potion but you cannot fly there therefore cannot explore it (I think).
